pvlib/temperature.py

```diff
@@ -0,0 +1,253 @@
+"""
+The ``temperature`` module contains functions for modeling temperature of
+PV modules and cells.
+"""
+
+import numpy as np
+
+
+TEMPERATURE_MODEL_PARAMETERS = {
+    'sapm': {
+        'open_rack_glass_glass': {'a': -3.47, 'b': -.0594, 'deltaT': 3},
+        'close_mount_glass_glass': {'a': -2.98, 'b': -.0471, 'deltaT': 1},
+        'open_rack_glass_polymer': {'a': -3.56, 'b': -.0750, 'deltaT': 3},
+        'insulated_back_glass_polymer': {'a': -2.81, 'b': -.0455, 'deltaT': 0},
+    },
```

Comment:
I see that some of the parameters were removed in the transition, probably for good reasons. Doesn't look like that change was documented, though.